### ISSUE TITLE
Failed to get VSCode command path

### ISSUE THREAD

**Issue by zcutlip** [takkaO/VSCode-OpenWindowTabContext#16]
I starting getting the error recently "Failed to get VSCode command path." I'm on version 1.68.0, on macOS.

I suspect a change to VSCode causes `process.execPath` to report:
`/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper.app/Contents/MacOS`

rather than:
`/Applications/Visual Studio Code.app/Contents/MacOS`

I was able to change `path.ts` to look in an alternate location, and it works:
```typescript
	// candidate 2
	let app_bin_path = "../Resources/app/bin";
	if (process.execPath.includes("Frameworks/Code Helper.app")) {
		app_bin_path = "../../../../Resources/app/bin";
	}
	code_path = path.join(path.dirname(process.execPath), app_bin_path, command);
```

Let me know if I can help in any way

```console
$ code --version
1.68.0
4af164ea3a06f701fe3e89a2bcbb421d2026b68f
arm64
$ sw_vers
ProductName:	macOS
ProductVersion:	12.4
BuildVersion:	21F79
```

**Comment by alinsavix** [comment]
I can confirm this problem is happening to me as well, with the same vscode/macOS versions.

In the meantime, setting `owtc.command` to the full path of the `code` executable is a functional workaround.

**Comment by takkaO** [comment]
I fixed this issue at version ```1.10.0``` and released it.
Please check and give feedback !

Thank you.

**Comment by zcutlip** [comment]
It's working for me on 1.10.0. Thank you! 👍👍
